syft/format/internal/cyclonedxutil/encoder.go

```diff
@@ -0,0 +1,37 @@
+package cyclonedxutil
+
+import (
+	"io"
+
+	"github.com/CycloneDX/cyclonedx-go"
+
+	"github.com/anchore/syft/syft/format/common/cyclonedxhelpers"
+	"github.com/anchore/syft/syft/sbom"
+)
+
+const DefaultVersion = "1.5"
```

Comment:
It seemed that the reason for this was because the json schema was not validated, however, at this time it is published. I might be misremembering here... do you recall a specific reason?

Reply:
I don't recall the specific reason; maybe @spiffcs does? The original PR that added support for versions in CDX defaulted to 1.4.

Reply:
confirmed on the v1.5 json schema not being available. Also the spec is minimally implemented, however, that's all we need.